share/containers/fedora.dockerfile

```diff
@@ -15,16 +15,19 @@ RUN ./autogen.sh \
 	--enable-shadowgrp \
 	--enable-man \
 	--with-audit \
+	--with-libpam \
 	--with-sha-crypt \
 	--with-bcrypt \
 	--with-yescrypt \
 	--with-selinux \
-	--without-libpam \
-	--enable-shared \
 	--without-libbsd \
+	--without-libcrack \
+	--without-sssd \
+	--enable-shared \
```

Comment:
Would you mind sorting `--enable-shared` in a separate commit from the one that changes flags?

Reply:
Also, there are more `--enable-*` flags further down.  I guess if we're sorting it, that would be the best place, no?

Reply:
This is what sort(1) produces:

```
	--disable-account-tools-setuid
	--enable-lastlog \
	--enable-logind=no \
	--enable-man \
	--enable-shadowgrp \
	--enable-shared \
	--with-audit \
	--with-bcrypt \
	--with-group-name-max-length=32 \
	--with-libpam \
	--with-selinux \
	--with-sha-crypt \
	--with-yescrypt \
	--without-libbsd \
	--without-libcrack \
	--without-sssd \
```

Reply:
Updated